tests/bookinfo/bookinfo.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+SCRIPTDIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+BOOKINFO_URL=https://raw.githubusercontent.com/istio/istio/release-1.0/samples/bookinfo
+
+echo
+
+if [ -z "$NAMESPACE" ]; then
+	NAMESPACE=default
+fi
+echo "export NAMESPACE=$NAMESPACE"
+
+if [ -z "$WITH_ISTIO" ]; then
+	WITH_ISTIO=false
+fi
+echo "export WITH_ISTIO=$HAS_ISTIO"
```

Comment:
do not understand the previous if statement before, seems to be overridden by this ?

Reply:
fixed to:

```
echo "export WITH_ISTIO=$WITH_ISTIO"
```

so that one can know what default settings were used